### PR TITLE
Toggle Throw Action when Grenade HandActivate

### LINKED ISSUE #1754: Triggering grenades doesnt active the throw action
# Description
When triggering your grenade on, your throw action won't be turned on, forcing the player to activate it manually

## Category
TODO

## Steps to Reproduce
Please enter the steps to reproduce the bug or behaviour:
1.Get a grenade
2.Activate it
3.Aim and click
4.Nothing happens as your throw action wasn't activated

### PR DESCRIPTION
**Pull request by christmoore**
Toggles the throw action when a grenade is interacted with.

Remove unused "explosion" string.

### Purpose
_Fixes #1754 to allow grenades thrown after activation without manual throw toggle_<br>

### Please make sure you have followed the self checks below before submitting a PR:

- [x] Code is sufficiently commented
- [x] Code is indented with tabs and not spaces
- [x] The PR does not include any unnecessary .meta, .prefab or <b>.unity (scene) changes</b>
- [x] The PR does not bring up any new compile errors
- [x] The PR has been tested in editor

Grenade

### CODE REVIEW

File: UnityProject/Assets/Scripts/Weapons/Grenade.cs [hunk 1]
```diff
@@ -96,6 +96,8 @@ private IEnumerator TimeExplode(GameObject originator)
 		{
 			timerRunning = true;
 			PlayPinSFX(originator.transform.position);
+			UIManager.Action.Throw();
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Weapons/Grenade.cs]: Only problem is this gets only called Server side so for Clients connecting to the server this is not triggered properly

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Weapons/Grenade.cs]: https://github.com/unitystation/unitystation/wiki/Implementing-Interactions#how-do-i-inform-the-client-what-happened 

Should explain it a little bit

Comment (Jibbajabbafic) [reply UnityProject/Assets/Scripts/Weapons/Grenade.cs]: Have a look at the `ExplodeWhenWelded` example in the IF2 docs. It explains what each function does and you should be able to implement something similar for the grenade: https://github.com/unitystation/unitystation/wiki/Implementing-Interactions#how-do-i-implement-an-interaction

**Comment by Jibbajabbafic** [comment]
Since this involves networking you should test it with at least 2 clients to make sure it works for the host and any clients connecting to the host

**Comment by christmoore** [comment]
Force push to remove previous site information. (Was fiddling with wiki stuff)

Works with clients. Thank you @chairbender, @Jibbajabbafic , and @Bod9001  for your help on this.

**Comment by christmoore** [comment]
Ready for merge

**Comment by christmoore** [comment]
Closing and reopening on a separate branch. Wiki changes require me to fiddle with develop branch, which this change is on.

**Pull request closed without merging**